### ISSUE TITLE
Crash after lot of page navigation

### ISSUE THREAD

**Issue by malat** [ariya/phantomjs#13556]
![58b081aa-d78f-41a8-b9b5-199384ac8ab6](https://cloud.githubusercontent.com/assets/3868621/9701905/b9fa6e04-5411-11e5-8776-90a5ecf79d11.jpg)

**Comment by malat** [comment]
Its seems to crash always after 243 recursive page.open

**Comment by malat** [comment]
each page.open is waiting to finish before starting the next one

**Comment by malat** [comment]
Do any one monitor issue in this project ???

**Comment by Vitallium** [comment]
@malat I do.

**Comment by zackw** [comment]
I'm sorry we took so long to get back to you about this.

With regret, I don't know if we're going to be able to do anything about this.  It's probably some kind of memory leak or memory corruption in the interaction between Qt and Webkit, and we don't have the manpower to debug it.  (I have debugged things like this before - it can take *days* of investigation.)

Personally, I would recommend quitting and restarting the PhantomJS process after every page load. I use PhantomJS as a scraper that way and it works well.

If *you* have time to debug this and work up a patch, though, we would be glad to take it.
